kubernetes/dev/server/external-secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: llm-ops-server-cluster-main
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: aws-secretsmanager
+  data:
+    - secretKey: llm-ops-secret
```

Comment:
Used here - https://github.com/smartnews/dp-LLM-ops/blob/2157e70d8184f23c6d51c2705831ddc0a73c3ced/kubernetes/dev/server/pezzo-server-deployment.yaml#L27-L36

Reply:
(Resume the discussion in here: https://smartnews.slack.com/archives/C07E5DUHX6F/p1734928433041409?thread_ts=1734689105.762669&cid=C07E5DUHX6F)

Maybe I mis-read. But I think redis cluster secret has different structure from clickhouse cluster secret. That's why you probably will need `property: password`